pkg/operator/enable_pacemaker_v1.go

```diff
@@ -0,0 +1,152 @@
+package operator
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/trento-project/workbench/internal/systemd"
+)
+
+const (
+	EnablePacemakerOperatorName = "enablepacemaker"
```

Comment:
nitpick:
I think `pacemakerenable` looks more in the line of the current operators, where we have `{resource}{action}` nomenclature